doc/examples/nonstationary_inference.rst

```diff
@@ -0,0 +1,87 @@
+.. jupyter-execute::
+    :hide-code:
+
+    import set_working_directory
+
+
+.. _nonstationary-model-aa-inference:
+
+
+Infering Amino Acid frequencies with a nonstationary model
+==========================================================
+
+.. sectionauthor:: Peter Goodman, Andrew Wheeler
+
+This example demonstrates how to apply a non-stationary, branch-heterogeneous maximum likelihood model to infer site frequency vectors across different branches or branch groups on a phylogenetic tree, highlighting a practical use case. The model used is a maximum likelihood, time-heterogeneous, and time-reversible model.
```

Comment:
- should that be "infer state frequency vectors"?
- "time-heterogeneous, and time-reversible model". Can you confirm this is time-reversible? (It is possible, but I just need to check with you. Noting that time reversible requires $$\pi_i Q_{i,j} = \pi_j Q_{j,i}$$